examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -136,22 +135,22 @@ As a result id of the transaction and id of the sidechain will be printed:
 ```
 {
     "txid": "9e4676274f1ff9b3164de6e0d6492c4dfc1d564b0243a36208c6b7fe848f9d21",
-    "scid": "2f7ed2e07ad78e52f43aafb85e242497f5a1da3539ecf37832a0a31ed54072c3",
+    "scid": "2f7ed2e07ad78e52f43aafb85e242497f5a1da3539ecf37832a0a31ed54072c3"
 }
 ```
 
 Parameters for `sc_create` are:
 1. Length of the withdrawal epoch (`10`) - how often (in Mainchain blocks size) backward transfers from sidechain to mainchain could be done
 2. Address to transfer initial coins amount (`"a5b...2ac"`) - It is public key was generated by `generatekey` command on step 3
 3. Initial coins amount (`100`)
-4. Verification key from step 4 (`"5e7...000"`) - That key is used for verifying backward transfers, verification file shall contains that key in byte form
+4. Verification key from step 4 (`"5e7...000"`) - That key is used for verifying backward transfers, verification file shall contain that key in byte form
 5. Vrf public key (`"dd2...000"`) - that key was generated by `generateVrfKey` command on step 3
 6. GenSysConstant (`"b86...000"`) - it was generated on step 4
 
 _Sidechain id will be generated automatically._
 
 **Step 8: Apply Sidechain declaration in a Mainchain node**
-Sidechain creation transaction now in memory pool of Mainchain node, thus Sidechain is prepared, but not created yet. For creating new Sidechain Sidechain creation transaction shall be put into Mainchain block. To do it generate 1 more block:  
+Sidechain creation transaction now in memory pool of Mainchain node, thus Sidechain is prepared, but not created yet. Sidechain creation transaction shall be put into Mainchain block. To do it generate 1 more block:
```

Comment:
and add next for Step into step description: "Skip that step, in case if you run your Sidechain in testnet or mainnet, because Mainchain block with declared Sidechain creation transaction will be mined automatically"